src/expandableCalendar/agendaList.tsx

```diff
@@ -142,7 +141,7 @@ const AgendaList = forwardRef((props: AgendaListProps, ref: any) => {
     return sectionTitle;
   }, []);
 
-  const scrollToSection = useCallback(debounce((d) => {
+  const scrollToSection = useCallback((d) => {
```

Comment:
As we discussed, removing the debounce only going to cause an issue with the list's updates from others source. Let's add the option to control this debounce time